content/app/guides/payment/frontend-configuration/_index.en.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Frontend configuration 
+description: Follow these steps to start developing apps in Altinn Studio.
+weight: 3
+---
+### 1. Add Payment layoutSet
+
+Add a new layoutSet folder for your payment task, and update your layout-sets.json.
+
+Your layout-sets.json may look something like this:
+
+```json
+{
+  "$schema": "https://altinncdn.no/toolkits/altinn-app-frontend/4/schemas/json/layout/layout-sets.schema.v1.json",
+  "sets": [
+    {
+      "id": "form",
+      "dataType": "model",
+      "tasks": [
+        "Task_1"
+      ]
+    },
+    {
+      "id": "payment",
+      "dataType": "model",
+      "tasks": [
+        "Task_2"
+      ]
+    }
+  ]
+}
+```
+
+In your payment layoutSet folder, add a new file, payment.json, with the following layout:
+
+```json
+{
+  "$schema": "https://altinncdn.no/toolkits/altinn-app-frontend/4/schemas/json/layout/layout.schema.v1.json",
+  "data": {
+    "layout": [
+      {
+        "id": "payment-test-component",
+        "type": "Payment",
+        "textResourceBindings": {
+          "title": "Oppsummering"
+        }
+      }
+    ]
+  }
+}
+```
+
+This is required for payment to work, without it, your payment step will just render a white page.
```

Comment:
This is required for payment to work. Without it, your payment step will just render a white page.